handlers/namespaces.go

```diff
@@ -28,7 +28,7 @@ func NamespaceMetrics(w http.ResponseWriter, r *http.Request) {
 	getNamespaceMetrics(w, r, prometheus.NewClient)
 }
 
-// NamespaceHealth is the API handler to get health of every services in the given namespace
+// NamespaceHealth is the API handler to get app-based health of every services in the given namespace
```

Comment:
I know I am missing tons of concepts in this area, but is it this endpoint based in both `app-based` and `workload-based` health?

Reply:
Yes, it's an endpoint used from the graph to get all health at once for all items in a namespace, "items" being either workloads or apps.